coalib/parsing/LineParser.py

```diff
@@ -62,8 +62,16 @@ def parse(self, line):
         # Escapes in value might be needed by the bears
         keys, value = self.__extract_keys_and_value(line)
 
+        all_delimiters = self.key_value_delimiters
+        all_delimiters += self.key_delimiters
+        all_delimiters += self.comment_seperators
+        all_delimiters += self.section_override_delimiters
+        all_delimiters = "".join(all_delimiters)
```

Comment:
not content with this yet, some of the delimiters are dicts, some are lists, can you test delims of each group please and maybe a comment why this works on dicts and lists?